src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file command";
+
+        public const string AppDescription = "Command to add, remove, and list projects from the solution (SLN) file.";
+
+        public const string AppHelpText = "Projects to add to the solution.";
```

Comment:
This should be the AppHelpText for the top-level sln command (it currently says "Projects to add to the solution."). Either that or you can remove and I'll use AppDescription for both description and help text.